src/@next/Steps/StepStyle.ts

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components';
+import { Breakpoints } from '..';
+import { space4, space12, space16 } from '../utilities/spacing';
+import { Blue, Neutral, Red } from '../utilities/colors';
+
+export const StepItemWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  gap: ${space12};
+  cursor: default;
+
+  &[data-dot='true'] {
+    gap: ${space16};
+  }
+
+  &[data-clickable='true'] {
+    cursor: pointer;
+  }
+`;
+
+export const CircleDiv = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 28px;
+  width: 28px;
+  border-radius: 50%;
+
+  svg {
+    height: 16px;
+    width: 16px;
+  }
+
+  &[data-variant='pending'] {
+    background-color: ${Neutral.B95};
+  }
+  &[data-variant='completed'] {
+    background-color: ${Blue.S08};
+    svg {
+      fill: ${Blue.S99};
+    }
+  }
+  &[data-variant='processing'] {
+    background-color: ${Blue.S99};
+  }
+  &[data-variant='error'] {
+    background-color: ${Red.B93};
+    svg {
+      fill: ${Neutral.B100};
+    }
+  }
+
+  &[data-dot='true'] {
+    height: 10px;
+    width: 10px;
+
+    > div,
+    > svg {
+      display: none;
+    }
+
+    &[data-variant='completed'] {
+      background-color: ${Blue.S99};
+    }
+  }
+
+  @media (max-width: ${Breakpoints.large}) {
+    height: 24px;
+    width: 24px;
+    svg {
+      height: 14px;
+      width: 14px;
+    }
+  }
+`;
+
+export const VerticalLineWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: ${space4} 0;
+  height: 64px;
+  width: 28px;
```

Comment:
@michac789 vertical line doesn't centrally align with the circle div on small screens (24px)